README.md

```diff
@@ -1,6 +1,7 @@
 [![Build status](https://img.shields.io/appveyor/ci/processhacker/processhacker.svg?style=for-the-badge)](https://ci.appveyor.com/project/processhacker/processhacker)
 [![Build contributors](https://img.shields.io/github/contributors/processhacker/processhacker.svg?style=for-the-badge&color=blue)](https://github.com/processhacker/processhacker/graphs/contributors)
 [![Licence](https://img.shields.io/badge/license-GPLv3-blue.svg?style=for-the-badge)](https://www.gnu.org/licenses/gpl-3.0.en.html)
+![Visitor count](https://shields-io-visitor-counter.herokuapp.com/badge?page=processhacker.processhacker&labelColor=000000&logo=GitHub&logoColor=FFFFFF&color=1D70B8&style=for-the-badge)

```

Comment:
This domain isn't `shields.io`. It is `heroku.com`. Which is owned by `SalesForce`. I believe information obtained through this counter will adhere to this policy: https://www.salesforce.com/company/privacy/. But, heroku provides a dashboard for accessing this information: https://data.heroku.com/.

I am unsure who owns that information and since we ultimately have no control over what data is collected or how it is used. I don't think we should use this.

In contrast `shields.io` is transparent and completely open source:
https://github.com/badges/shields
https://metrics.shields.io/d/g_1B7zhik/prom-client-default-metrics?orgId=1&refresh=10s

Reply:
moved this comment to review feedback